src/main/rx/rx.c

```diff
@@ -396,6 +405,15 @@ static void setLinkQuality(bool validFrame, timeDelta_t currentDeltaTime)
     }
 }
 
+void setLinkQualityDirect(uint16_t linkqualityValue)
+{
+    UNUSED(linkqualityValue);
```

Comment:
Should go into the `#else`.

Reply:
ok .. will do 

#ifndef USE_RX_LINK_QUALITY_INFO
    UNUSED(xxxxx);
#else
....
like in  adc